pkg/subnet/local_test.go

```diff
@@ -78,7 +78,7 @@ func TestDeployToLocal(t *testing.T) {
 	err = os.WriteFile(testGenesis.Name(), []byte(genesis), constants.DefaultPerms755)
 	assert.NoError(err)
 	// test actual deploy
-	err = testDeployer.DeployToLocalNetwork("test", testGenesis.Name())
+	_, _, err = testDeployer.DeployToLocalNetwork("test", testGenesis.Name())
```

Comment:
Do we not want to check the output in this test?

Reply:
This isn't really deploying anything, the returned ids are just empty IDs. But we can test for those just to not leave this look "unfinished".

Reply:
This isn't a huge deal, but could we inject a non-empty value via a mock or whatever so we can actually check the return is set properly? If it's too complicated, we can pass.

Reply:
I have edited the existing test to account for this.